docs/notebooks/image_to_image_arcface.ipynb

```diff
@@ -0,0 +1,362 @@
+        "\n",
+        "finetuner.login(force=True)"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "ONpXDwFBsqQS"
+      },
+      "outputs": [],
+      "source": [
+        "train_data = DocumentArray.pull('finetuner/stanford-cars-train', show_progress=True)\n",
+        "query_data = DocumentArray.pull('finetuner/stanford-cars-query', show_progress=True)\n",
+        "index_data = DocumentArray.pull('finetuner/stanford-cars-index', show_progress=True)\n",
+        "\n",
+        "train_data.summary()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "mUoY1jq0klwk"
+      },
+      "source": [
+        "## Backbone model\n",
+        "Now let's see which backbone models we can use. You can see available models by calling `finetuner.describe_models()`.\n",
+        "\n",
+        "\n",
+        "For this example, we're gonna go with `resnet50`."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "xA7IIhIOk0h0"
+      },
+      "source": [
+        "## Fine-tuning\n",
+        "\n",
+        "Now that we have the training and evaluation datasets loaded as `DocumentArray`s and selected our model, we can start our fine-tuning run."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "qGrHfz-2kVC7"
+      },
+      "outputs": [],
+      "source": [
+        "from finetuner.callback import EvaluationCallback\n",
+        "\n",
+        "run = finetuner.fit(\n",
+        "    model='resnet50',\n",
+        "    train_data='finetuner/stanford-cars-train',\n",
+        "    batch_size=128,\n",
+        "    epochs=5,\n",
+        "    learning_rate=1e-3,\n",
+        "    loss='ArcFaceLoss',\n",
+        "    device='cuda',\n",
+        "    sampler='random',\n",
+        "    callbacks=[\n",
+        "        EvaluationCallback(\n",
+        "            query_data='finetuner/stanford-cars-query',\n",
+        "            index_data='finetuner/stanford-cars-index',\n",
+        "        )\n",
+        "    ],\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "9gvoWipMlG5P"
+      },
+      "source": [
+        "Let's understand what this piece of code does:\n",
+        "\n",
+        "* As you can see, we have to provide the `model` which we picked before.\n",
+        "* We also set `run_name` and `description`, which are optional,\n",
+        "but recommended in order to retrieve your run easily and have some context about it.\n",
+        "* Furthermore, we had to provide names of the `train_data`.\n",
+        "* We set `ArcFaceLoss`.\n",
+        "* Additionally, we use `finetuner.callback.EvaluationCallback` for evaluation.\n",
+        "* Lastly, we set the number of `epochs` and provide a `learning_rate`."
```

Comment:
```suggestion
        "Let's understand what this piece of code does:\n",
        "\n",
        "* We select a `model`: `resnet50`.\n",
        "* We set `run_name` and `description`, which are optional,\n",
        "but strongly recommended so that you can access and retain information about your run.\n",
        "* We specify the training data (`train_data`).\n",
        "* We set `ArcFaceLoss` as our loss function.\n",
        "* We use `finetuner.callback.EvaluationCallback` for evaluation and specify the query and index datasets for it.\n",
        "* We set the number of training epochs (`epochs`) and the learning rate (`learning_rate`)."
```